scripts/falco-driver-loader

```diff
@@ -426,70 +432,40 @@ load_bpf_probe_download() {
 	echo "* Trying to download a prebuilt eBPF probe from ${URL}"
 
 	if ! curl -L --create-dirs "${FALCO_DRIVER_CURL_OPTIONS}" -o "${HOME}/.falco/${BPF_PROBE_FILENAME}" "${URL}"; then
-		>&2 echo "Download failed"
-		exit 1;
+		>&2 echo "Unable to find a prebuilt Falco eBPF probe"
+		return
 	fi
 }
 
 load_bpf_probe() {
-
-	echo "* Mounting debugfs"
-
-	if [ ! -d /sys/kernel/debug/tracing ]; then
-		mount -t debugfs nodev /sys/kernel/debug
-	fi
-
-	get_kernel_config
-
-	if [ -n "${HOST_ROOT}" ] && [ -f "${HOST_ROOT}/etc/os-release" ]; then
```

Comment:
I don't think this has anything to do with the inversion of download vs compile here